### ISSUE TITLE
Getting tooltips to show on mobile without clicking link

### ISSUE THREAD

**Issue by julianguyen** [julianguyen/ifme#205]
I recently added stats on Categories, Moods, and Strategies on user profiles. In your own profile, you can hover over each stat and a tooltip with a list of viewers will show up. On mobile web, you can't hover... so the only way you can see it is if you tap the text... but tapping the text will click the link. 

![screen shot 2016-06-04 at 11 31 32 am](https://cloud.githubusercontent.com/assets/3010728/15801411/7e979ae0-2a48-11e6-8cbe-e8510c6e7a17.png)

Instead, we want to be able to tap over anywhere in the background div and have the viewers tooltip show up.

**Comment by julianguyen** [comment]
Please look at viewers_hover in application_controller.rb :)

**Comment by andela-cdaniel** [comment]
Hi, good day. 

I got to know of this project from `yourfirstpr` on `twitter` and I'd like to have a go at this issue. 

I've already hacked at it from the browser and was able to come up with this:

<img width="1280" alt="screen shot 2016-06-05 at 10 30 44 am" src="https://cloud.githubusercontent.com/assets/7131935/15804678/3279547a-2b09-11e6-86ed-9708411dd7c3.png">
<img width="1280" alt="screen shot 2016-06-05 at 10 31 05 am" src="https://cloud.githubusercontent.com/assets/7131935/15804679/3284114e-2b09-11e6-9430-162cde146ffb.png">
<img width="1280" alt="screen shot 2016-06-05 at 10 31 24 am" src="https://cloud.githubusercontent.com/assets/7131935/15804680/329ed718-2b09-11e6-908b-c5fe4afb2ac7.png">

The idea behind it is to only show the modified `tooltip` when on mobile and make it dismissible. Doing it this way ensures that the default behaviour of clicking a link and having it redirect to a different page is preserved, furthermore, the user will not have to bother about clicking the text or the background encompassing the text.
